platform-core-module/src/test/java/com/softicar/platform/core/module/page/PageDivTest.java

```diff
@@ -85,8 +85,21 @@ public void testOpeningPageWithParameters() {
 	}
 
 	@Test
-	public void testMultipleOpenFolder() {
+	public void testMultipleOpenFolderWithCollapsing() {
 
```

Comment:
For some extra clarity, we could add `testUser.setCollapseNavigationFolders(true).save();` here